ext/README.md

```diff
@@ -0,0 +1,23 @@
+# Ext
+
+This gem dependes on the `librdkafka` C library. It is downloaded when
+this gem is installed.
+
+To update the `librdkafka` version follow the following steps:
+
+* Download the new version `tar.gz` from  https://github.com/edenhill/librdkafka/
+* Generate a `sha256` with (if using MacOS) `shasum -a <file>`
```

Comment:
You need to specify the algorithm: `shasum -a 256 <file>`

Reply:
fixed